app/serializers/important_events_serializer.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class ImportantEventsSerializer
+  include JSONAPI::Serializer
+
+  set_type :events
+
+  attributes :event_type, :classification
```

Comment:
Are we 100% sure we don't more attributes than this?

Reply:
We need the event's id, that's for sure

Reply:
That comes automatically because of JSONAPI